model/src/w3sic5md.F90

```diff
@@ -1,5 +1,27 @@
+!> @file
+!> @brief Calculate ice source term S_{ice} according to different ice
+!>  models.
+!>
+!> @author Q. Liu
+!> @author E. Rogers
+!> @date   19-May-2021
+!>
+
 #include "w3macros.h"
 !/ ------------------------------------------------------------------- /
+!>
+!> @brief Calculate ice source term S_{ice} according to different ice
+!>     models:
+!>
+!> @author Q. Liu
+!> @author E. Rogers
+!> @date   19-May-2021
+!>
```

Comment:
any reason: 

  !     * 'FS': the viscoelastic, extended Fox and Squire sea ice model
  !             (Mosig et al. 2015)
  !     * 'RP': the viscoelastic, Robinson and Palmer model (Mosig et al.
  !             2015)
  !     * 'M2': the order 3 power law model proposed by Meylan et al.
  !             (2018)
  With references are not in this section here?